data-host/library/src/popup/services/StorageService.test.ts

```diff
@@ -0,0 +1,310 @@
+import { StorageService } from '@services/StorageService';
+import {
+  StorageServiceClear as ERROR_CLEAR,
+  StorageServiceGetItem as ERROR_GET_ITEM,
+  StorageServiceHasItem as ERROR_HAS_ITEM,
+  StorageServiceRemoveItem as ERROR_REMOVE_ITEM,
+  StorageServiceSetItem as ERROR_SET_ITEM,
+} from '@services/StorageService';
+import { chrome } from 'jest-chrome';
+
+console.error = jest.fn();
```

Comment:
We shouldn't override console.error. If we do, we should do it just for one test and then reset back to what it was.